ShowPSAst.psm1

```diff
@@ -1,255 +1,23 @@
-            ColumnHeadersVisible = $true
-            Font = $font
-            Location = [System.Drawing.Point]@{X = 12; Y = 424}
-            ReadOnly = $true;
-            RowHeadersVisible = $false
-            SelectionMode = [System.Windows.Forms.DataGridViewSelectionMode]::FullRowSelect
-            Size = [System.Drawing.Size]@{Width = 600; Height = 256}
-            TabIndex = 1
-        }
-        $dataView.Columns.AddRange(
-            [System.Windows.Forms.DataGridViewTextBoxColumn]@{
-                HeaderText = 'Property'        
-                ReadOnly = $true
-                AutoSizeMode = [System.Windows.Forms.DataGridViewAutoSizeColumnMode]::AllCellsExceptHeader},
-            [System.Windows.Forms.DataGridViewTextBoxColumn]@{
-                HeaderText = 'Value'
-                ReadOnly = $true
-                Resizable = [System.Windows.Forms.DataGridViewTriState]::True
-                AutoSizeMode = [System.Windows.Forms.DataGridViewAutoSizeColumnMode]::Fill},
-            [System.Windows.Forms.DataGridViewTextBoxColumn]@{
-                HeaderText = 'Type'
-                ReadOnly = $true
-                AutoSizeMode = [System.Windows.Forms.DataGridViewAutoSizeColumnMode]::AllCellsExceptHeader
-        })
-
-        # The script view is a text box that displays the text of the script.
-        # If the text box has not been edited, selecting an ast in the tree view
-        # will select the matching text in the script view.
-        $scriptView = [System.Windows.Forms.TextBox]@{
-            Font = $font
-            HideSelection = $false
-            Location = [System.Drawing.Point]@{X = 624; Y = 12}
-            Multiline = $true
-            ScrollBars = 'Both'
-            Size = [System.Drawing.Size]@{Width = 561; Height = 668}
-            TabIndex = 2
-            Text = $InputObject.Extent.Text
-            WordWrap = $false
-        }
-
-        $script:BufferIsDirty = $false
-        $scriptView.Add_TextChanged({ $script:BufferIsDirty = $true })
-        $scriptView.Add_KeyUp({ OnTextBoxKeyUp @args })
-
-        $script:inputObjectStartOffset = $InputObject.Extent.StartOffset
-
-        try
-        {
-            # Create the main form and show it.
-            $form = [Windows.Forms.Form]@{
-                Text = "Ast Explorer"
-                ClientSize = [System.Drawing.Size]@{Width = 1200; Height = 700}
-            }
-            $form.Controls.Add($dataView)
-            $form.Controls.Add($treeView)
-            $form.Controls.Add($scriptView)
-            $null = $form.ShowDialog()
-        } finally {
-            $form.Dispose()
-        }
+#Dot source the files
+Foreach ($import in @($public + $private)) {
+    Try {
+        . $import.fullname
+    }
+    Catch {
+        Write-Error -Message "Failed to import function $($import.fullname): $_"
     }
 }
 
+$showPsAstConfig = Get-ShowPsAstConfig
```

Comment:
I think defaults should be loaded as needed so changes to the configuration that happen after module load are honored.